metrics/mixin/alerts/storage-client.libsonnet

```diff
@@ -0,0 +1,69 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'storage-client-alerts.rules',
+        rules: [
+          {
+            alert: 'StorageClientHeartbeatWarn',
+            expr: |||
+              time() - %(clientCheckinWarnSec)d > ocs_storage_client_last_heartbeat > 0
+            ||| % $._config,
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              message: 'Storage Client ({{ $labels.storage_consumer_name }}) heartbeat missed for more than %d (s)' % $._config.clientCheckinWarnSec,
+              description: 'Storage Client ({{ $labels.storage_consumer_name }}) heartbeat missed for more than %d (s). Lossy network connectivity might exist' % $._config.clientCheckinWarnSec,
+              severity_level: 'warning',
+            },
+          },
+          {
+            alert: 'StorageClientHeartbeatCrit',
+            expr: |||
+              time() - %(clientCheckinWarnSec)d > ocs_storage_client_last_heartbeat > 0
+            ||| % $._config,
+            labels: {
+              severity: 'critical',
+            },
+            annotations: {
+              message: 'Storage Client ({{ $labels.storage_consumer_name }}) heartbeat missed for more than %d (s)' % $._config.clientCheckinWarnSec,
```

Comment:
I think you want here to use `clientChekinCritSecs`

Reply:
Yep, missed it.